src/ocl/tensorocl.cpp

```diff
@@ -214,21 +214,21 @@ void OpTensor3d(const Handle& handle,
                 auto kernel = kernels.front();
 
                 kernel(ATensor,
-                       int(astrides[1]), // a_cstride,
+                       static_cast<int>(astrides[1]), // a_cstride,
                        BTensor,
-                       int(bstrides[1]), // b_cstride,
+                       static_cast<int>(bstrides[1]), // b_cstride,
                        CTensor,
-                       int(cstrides[1]), // c_cstride,
+                       static_cast<int>(cstrides[1]), // c_cstride,
                        miopen_alpha0,
                        miopen_alpha1,
                        miopen_beta,
-                       long(Aoffset),
-                       long(Boffset),
-                       long(Coffset),
-                       long(total_work),
-                       long(total_work2),
-                       int(!float_equal(miopen_beta, 0.0)),
-                       int(blens[1] == 1));
+                       static_cast<int64_t>(Aoffset),
```

Comment:
[Notice] Formally, here we should use `cl_long` (OCL way), or `long long` (HIP way). `int64_t` would also work, of course.